src/ops/kernels/concat_kernels.cu

```diff
@@ -13,7 +13,7 @@
  * limitations under the License.
  */
 
-#include "flexflow/ops/concat.h"
+#include "flexflow/ops/kernels/concat_kernels.h"
 #include "flexflow/utils/cuda_helper.h"
 #include "flexflow/utils/hash_utils.h"
```

Comment:
Why necessary?

Reply:
This is for the kernel function implementations, so we should include that header file. However, we don't need any of the methods in `concat.h`.

Reply:
I think I meant the `hash_utils` here :slightly_smiling_face:

Reply:
Removed.